### PR TITLE
[Docs] Add slow queries dashboard information into Platform docs

### PR DESCRIPTION
**Pull request by yb-andrew**
@sshev This PR also fixes an issue with footer not sitting at the bottom of the page when there is a small amount of content on the page. See this page for an example of what I mean: https://docs.yugabyte.com/latest/yugabyte-platform/alerts-monitoring/

### CODE REVIEW

File: docs/content/latest/yugabyte-platform/alerts-monitoring/slow-queries-dashboard.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+headerTitle: Slow Queries dashboard
```
Comment (chirag-yb) [inline docs/content/latest/yugabyte-platform/alerts-monitoring/slow-queries-dashboard.md:2]: @yb-andrew Please include these pointers?
1. Search and query language supported
2. On ramp from other pages 

Similar to what we have documented here (point 3,4,5) - https://docs.yugabyte.com/latest/yugabyte-platform/alerts-monitoring/live-queries-dashboard/

**Review by sshev — APPROVED** [review]

**Comment by polarweasel** [comment]
@yb-andrew do you need any help with this one, or what's the status of this PR?

**Comment by yb-andrew** [comment]
@polarweasel Will be adding the changes later today to the PR.

File: docs/content/latest/yugabyte-platform/alerts-monitoring/slow-queries-dashboard.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+Use filtering for comparisons on numbers columns (`Avg Time`) using `>`, `>=`, `<`, and `<=` to search for values that are greater than, greater than or equal to, less than, and less than or equal to another value (`Avg Time: < 30`). You can also use the range syntax `n..m` to search for values within a range, where the first number `n` is the lowest value and the second number `m` is the highest value. The range syntax supports tokens like the following: `n..*` which is equivalent to `>= n`. Or `*..n` which is the same as `<= n`.
+
+5. Click on a row to open a sidebar with a full view of the query statement, along with all the column data.
+
+![View query statement](/images/yp/alerts-monitoring/slow-queries/query-info-panel.png)
+
+You can also find additional prefiltered navigation links from different pages to the **Slow Queries** page.
+
+* Example: From the **Overview** tab to the **Queries** tab, when the user clicks the link to "Top SQL Statements".
```
Comment (lizayugabyte) [inline docs/content/latest/yugabyte-platform/alerts-monitoring/slow-queries-dashboard.md:70]: For example, if you click the link to "Top SQL Statements", you navigate from **Overview** to **Queries**.

File: docs/content/latest/yugabyte-platform/alerts-monitoring/slow-queries-dashboard.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+![Overview page showing slow queries](/images/yp/alerts-monitoring/slow-queries/overview-showing-link.png)
+
+* Example: The **Nodes** page each node's Actions contains a link to the **Slow Queries** page.
```
Comment (lizayugabyte) [inline docs/content/latest/yugabyte-platform/alerts-monitoring/slow-queries-dashboard.md:74]: Or, for example, on the **Nodes** page, each node's Actions contains a link to the **Slow Queries** page.

**Review by lizayugabyte — COMMENTED** [review]
Made some comments

**Pull request merged**